Sources/Components/Broadcast/BroadcastItem.swift

```diff
@@ -139,3 +139,9 @@ extension BroadcastItem: UITextViewDelegate {
         return false
     }
 }
+
+// Helper function inserted by Swift 4.2 migrator.
+fileprivate func convertToOptionalNSAttributedStringKeyDictionary(_ input: [String: Any]?) -> [NSAttributedString.Key: Any]? {
```

Comment:
This function looks weird 😅 Should we perhaps change `BroadcastItem.Style.linkTextAttributes` to `[NSAttributedStringKey.foregroundColor: UIColor.primaryBlue]` (without rawValue)?

Reply:
wow, Xcode!